File: src/main/java/no/nav/pensjon/regler/domain/beregning/AfpTillegg.java
package no.nav.pensjon.regler.domain.beregning;

import no.nav.pensjon.regler.domain.kode.YtelsekomponentTypeCti;

public class AfpTillegg extends Ytelseskomponent {

    private static final long serialVersionUID = -8986405690692738433L;

    public AfpTillegg() {
        super();
        ytelsekomponentType = new YtelsekomponentTypeCti("AFP_T");
    }

    public AfpTillegg(AfpTillegg afpTillegg) {
        super(afpTillegg);
        ytelsekomponentType = new YtelsekomponentTypeCti("AFP_T");
    }
}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning/AfpTillegg.kt
package no.nav.pensjon.regler.domain.beregning

import no.nav.pensjon.regler.domain.enum.YtelseskomponentTypeEnum

class AfpTillegg : Ytelseskomponent() {
    override val ytelsekomponentTypeEnum: YtelseskomponentTypeEnum = YtelseskomponentTypeEnum.AFP_T
}